source/class/qx/core/Object.js

```diff
@@ -162,9 +162,14 @@ qx.Class.define("qx.core.Object",
      */
```

Comment:
Please can you add to the method comment, reminding that `this.base` is still required even with the compiler because there are occasions such as `compile.js` or playground where the compiler is not used

Reply:
sure. Done